pkg/acsengine/engine.go

```diff
@@ -481,6 +487,11 @@ func (t *TemplateGenerator) getTemplateFuncMap(properties *api.Properties) map[s
 				return ""
 			}
 
+			// add artifacts and addons
```

Comment:
No it's needed here, but I thought you were right at first till expanded the code in Github. It's once used for the Master and once for the Agent.